runtime/oti/j9port.h

```diff
@@ -170,8 +170,10 @@
 #define J9PORT_SHSEM_MODE_NOWAIT ((uintptr_t) 2)
 /** @} */
 
-/* The following defines are used by j9shmem and j9shsem */
+/* The following defines are used by j9shmem.c, j9shsem.c and j9shsem_deprecated.c */
 #define J9SH_MAXPATH EsMaxPath
+#define MUTEX_STR "_mutex"
```

Comment:
Should be J9SH_MUTEX_STR and J9SH_SET_STR to be consistent with the other defines and help qualify their purpose.

Reply:
Fixed